docs/source/markdown/podman-machine-info.1.md

```diff
@@ -0,0 +1,36 @@
+% podman-machine-info(1)
+
+## NAME
+podman\-machine\-info - Display machine host info
+
+## SYNOPSIS
+**podman machine info**
+
+## DESCRIPTION
+
+Display information pertaining to the machine host.
+Rootless only.
```

Comment:
Can you elaborate on this? "Can only be used with rootless Podman as `podman machine` can only be used with rootless Podman" or similar?

Reply:
Sure